test/unit/cameras/PerspectiveCamera.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @author simonThiele / https://github.com/simonThiele
+ */
+
+module( "PerspectiveCamera" );
+
+test( "updateProjectionMatrix", function() {
+	var near = 1,
+			far = 3,
+			bottom = -1,
+			top = 1,
```

Comment:
I introduced them to check if the matrix calculation is done right. If you define a projection matrix by hand, you need t, b, l, r, n, f so I defined them manually and check the result agains them.